README.md

```diff
@@ -338,3 +338,22 @@ the [kubeflow.yaml](./kubeflow.yaml) file, you can edit the root
 [kustomization.yaml](./kustomization.yaml) and comment out the regular
 Volumes Web App and uncomment the PVCViewer Controller and Experimental
 Volumes Web App.
+
+## Troubleshooting
+
+### I can't get letsencrypt to work. The cert-manager logs show 404 errors.
+The `letsencrypt` HTTP-01 challenge is incompatible with using OIDC ([Link](https://www.jetstack.io/blog/istio-oidc/)). If your DNS server allows programmatic access, use the [DNS-01](https://cert-manager.io/docs/configuration/acme/dns01/) challenge solver instead.
+
+### I am having problems getting the deployment to run on a cluster deployed with kubeadm and/or kubespray.
```

Comment:
These flags are set by default with Kubernetes 1.20. Personally, that is the version I recommend running.